<launch>
  <arg name="audio_topic" />
  <arg name="image_topic" />
  <arg name="queue_size" />
  <arg name="do_timestamp" />
  <arg name="file_location" />
  <arg name="file_format" />
  <arg name="audio_channels" />
  <arg name="audio_sample_rate" />
  <arg name="audio_format" />
  <arg name="audio_sample_format" />

  <node name="audio_video_recorder" pkg="audio_video_recorder"
        type="audio_video_recorder" output="screen">
    <remap from="~input/audio" to="$(arg audio_topic)" />
    <remap from="~input/image" to="$(arg image_topic)" />
    <rosparam subst_value="true">
      queue_size: $(arg queue_size)
      do_timestamp: $(arg do_timestamp)
      file_location: $(arg file_location)
      file_format: $(arg file_format)
      audio_channels: $(arg audio_channels)
      audio_sample_rate: $(arg audio_sample_rate)
      audio_format: $(arg audio_format)
      audio_sample_format: $(arg audio_sample_format)
    </rosparam>
  </node>
</launch>
